hadoop-ozone/dist/src/main/compose/ozonesecure/docker-config

```diff
@@ -41,6 +41,12 @@ OZONE-SITE.XML_ozone.scm.client.address=scm
 OZONE-SITE.XML_hdds.block.token.enabled=true
 OZONE-SITE.XML_hdds.container.token.enabled=true
 OZONE-SITE.XML_hdds.grpc.tls.enabled=true
+
+OZONE-SITE.XML_hdds.x509.default.duration=PT40s
+OZONE-SITE.XML_hdds.x509.renew.grace.duration=PT30s
+OZONE-SITE.XML_hdds.block.token.expiry.time=15s
+OZONE-SITE.XML_ozone.manager.delegation.token.max-lifetime=15s
```

Comment:
15 seconds may be too short for the token tests to pass:

```
TOKEN_EXPIRED testuser tried to renew an expired token (OzoneToken owner=testuser/scm@EXAMPLE.COM, renewer=testuser, realUser=, issueDate=2023-05-09T14:28:19.801Z, maxDate=2023-05-09T14:28:34.801Z, sequenceNumber=1, masterKeyId=131, strToSign=null, signature=null, awsAccessKeyId=null, omServiceId=omServiceIdDefault, omCertSerialId=2513321810675) max expiration date: 2023-05-09 14:28:34,801+0000 currentTime: 2023-05-09 14:28:36,793+0000 |  
```

https://github.com/apache/ozone/actions/runs/4923738644/jobs/8802170708?pr=4681#step:5:623

Does token expiry matter for the cert rotation tests, anyway?

Reply:
It does not matter directly, but we have a sanity check in [StorageContainerManager](https://github.com/apache/ozone/blob/e8fb1744e63b0f87379f7cb0eb1cb8a98520bd95/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java#L951)
checking if the block token expiry time is shorter than the certificate renew grace duration, therefore we need to also change it here. I'm not sure how to avoid flakiness here other than increasing the token expiry time.

Reply:
One option is to add an undocumented config property that disables the sanity check you mention.  An example of such a config property:

https://github.com/apache/ozone/blob/e8fb1744e63b0f87379f7cb0eb1cb8a98520bd95/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java#L268-L272

https://github.com/apache/ozone/blob/e8fb1744e63b0f87379f7cb0eb1cb8a98520bd95/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestOzoneConfigurationFields.java#L88-L90